flink-formats/flink-csv/src/test/java/org/apache/flink/formats/csv/CsvRowDataSerDeSchemaTest.java

```diff
@@ -109,6 +109,54 @@ public void testSerializeDeserialize() throws Exception {
 			new byte[] {107, 3, 11});
 	}
 
+	@Test
+	public void testSerializeDeserializeForTime() throws Exception {
+		testFieldForTime(
+			TIME(3),
+			"12:12:12.232",
+			"12:12:12.232",
+			(deserSchema) -> deserSchema.setNullLiteral("null"),
+			(serSchema) -> serSchema.setNullLiteral("null"),
```

Comment:
Why not reuse the `testNullableField` method?

Reply:
For example, the input is "12:12:!2.232", but the precision is 2, then the result should be "12:12:12.23". The existing test named `testNullableField ` must be modified to meet the needs of this test. But the change affects other test cases. So i add a new test methed named `testFieldForTime`.

Reply:
For such case, I think you can use the test method only validate the deserialization.

```
testField(
			DataType fieldType,
			String csvValue,
			Object value,
			Consumer<CsvRowDataDeserializationSchema.Builder> deserializationConfig,
			String fieldDelimiter)
```

Reply:
Sorry for my carelessness, I will optimize it.

Reply:
@wuchong 
the following code 
`Row actualRow = (Row) DataFormatConverters.getConverterForDataType(dataType).toExternal(deserializedRow); `
in methed 
`testField(
			DataType fieldType,
			String csvValue,
			Object value,
			Consumer<CsvRowDataDeserializationSchema.Builder> deserializationConfig,
			String fieldDelimiter)`
will change TIME(n) (0<=n<=3) to  TIME(0) .Then the test will fail.
 I still use `testFieldForTime()`.

Reply:
This works in my environment:

```java
		testField(
			TIME(3),
			"12:12:12.232421",
			LocalTime.parse("12:12:12.232"),
			(deserSchema) -> deserSchema.setNullLiteral("null"),
			",");
```

Reply:
fix it